build-in-container/scripts/setup.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+echo "------------ Setting up Mariner Build Environment ------------"
+
+# build variables
+MARINER_BASE_DIR=/mariner
+BUILD_DIR="$MARINER_BASE_DIR/build"
+CCACHE_DIR="$MARINER_BASE_DIR/ccache"
+CBL_MARINER_GIT_URL="https://github.com/microsoft/CBL-Mariner.git"
+CHROOT_DIR="/temp/DockerStage/"
+DISABLE_UPSTREAM_REPOS="n"
+IMAGE_CONFIG_DIR="$MARINER_BASE_DIR/toolkit/imageconfigs"
+LOG_LEVEL="info"
+LOG_PUBLISH_DIR="$MARINER_BASE_DIR/logs"
+MARINER_RELEASE_TAG="2.0-stable"
+OUT_DIR="$MARINER_BASE_DIR/out"
+RUN_CHECK="y"
+SOURCE_URL="https://cblmarinerstorage.blob.core.windows.net/sources/core"
+SPECS_DIR="$MARINER_BASE_DIR/SPECS"
+USE_CCACHE="y"
+
+# Build Mariner toolkit if not present, by cloning Mariner GitHub repo
+#
+# No arguments
+# Global variables expected to be defined: BUILD_DIR, CHROOT_DIR, CHROOT_NB, OUT_DIR
+download_mariner_toolkit() {
+    if [ ! "$(ls -A toolkit)" ]; then
```

Comment:
```suggestion
   if [ ! -d "toolkit" ]; then
```

Reply:
The toolkit folder is created by the script already. This checks if the folder is empty or not

Reply:
ok, fair enough.